app/hotels/src/singleHotel/description/Description.js

```diff
@@ -48,11 +49,11 @@ const Link = ({
 );
 
 const renderTruncatedFooter = (handlePress: () => void) => (
-  <Link label="Show More" handlePress={handlePress} />
+  <Link label={TransformButtonText('Show More')} handlePress={handlePress} />
```

Comment:
Does this issue apply even to these links? I am not sure about that. I thought it's only about real buttons?